src/erlfmt_scan.erl

```diff
@@ -166,10 +168,14 @@ eof(undefined, Loc) ->
     {{eof, Loc}, undefined}.
 
 -spec last_node_string(state()) -> {unicode:chardata(), anno()}.
+last_node_string(#state{original = [{'#', _}, {'!', _} | _] = Tokens}) ->
+    split_shebang(Tokens);
```

Comment:
Why do we need this special case here?

Reply:
You are right, I needed it before, but in this new version, I don't need it anymore.  Good catch, I removed it.